website/profile/views.py

```diff
@@ -450,18 +450,22 @@ def user_choose_mailing_lists(auth, **kwargs):
     json_data = escape_html(request.get_json())
     if json_data:
         for list_name, subscribe in json_data.items():
-            update_subscription(user, list_name, subscribe)
+            # TO DO: change this to take in any potential non-mailchimp, something like try: update_subscription(), except IndexNotFound: update_mailchimp_subscription()
```

Comment:
It might be a good idea to make an 'improvement' ticket in Jira that 'depends' on the ticket this PR is assigned do. 

Also, as above, update (or make) the test that verifies this behavior.